tests/epyccel/test_epyccel_sets.py

```diff
@@ -0,0 +1,48 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+import pytest
+from  pyccel.epyccel import epyccel
+
+@pytest.fixture( params=[
+        pytest.param("fortran", marks = [
+            pytest.mark.skip(reason="set methods not implemented in fortran"),
+            pytest.mark.fortran]),
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="set methods not implemented in c"),
+            pytest.mark.c]),
+        pytest.param("python", marks = pytest.mark.python)
+    ],
+    scope = "module"
+)
+def language(request):
+    return request.param
+
+def test_add_1(language) :
+    def add_int():
```

Comment:
We should give a better name to the test functions. Here I would call it `test_add_literal_int`.

Reply:
fixed : 0ff444f